gulpfile.babel.js

```diff
@@ -1,18 +1,45 @@
 import gulp from 'gulp';
+import browserSync from 'browser-sync';
 import requireDir from 'require-dir';
-import livereload from 'gulp-livereload';
 
 requireDir( './gulp-tasks' );
 
+// We store some config objects here. So, let's load:
+const packageJson = require('../../../package.json');
+// Create a BrowserSync instance:
+const bs = browserSync.create();
+const proxyUrl = packageJson.browserSync.proxyUrl;
+
+gulp.task( 'bs-reload-css', ( cb ) => {
+	bs.reload('*.css');
+	cb();
+});
+
+gulp.task( 'bs-reload', ( cb ) => {
+	bs.reload();
+	cb();
+});
+
 gulp.task( 'js', gulp.series( 'webpack' ) );
 
 gulp.task( 'cssprocess', gulp.series( 'css', 'cssnano', 'cssclean' ) );
 
 gulp.task( 'watch', () => {
 	process.env.NODE_ENV = 'development';
-	livereload.listen( { basePath: 'dist' } );
-	gulp.watch( ['./assets/css/**/*.css', '!./assets/css/src/**/*.css'], gulp.series( 'cssprocess' ) );
-	gulp.watch( './assets/js/**/*.js', gulp.series( 'js' ) );
+
+	if ( proxyUrl ) {
+		// https://browsersync.io/docs/options
+		bs.init({
+			proxy: packproxyUrl,
```

Comment:
Hey, I think you've got a wee typo on `packproxyUrl`

Reply:
omg

Reply:
Mate if that's the worst bug you make in a PR you're doing well.